<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthorInformationClass</class>
 <widget class="QMainWindow" name="AuthorInformationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>330</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>431</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>431</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AuthorInformation</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnSignature">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>210</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="AuthorInformation.qrc">
      <normaloff>:/AuthorInformation/res/thatboy-text-vector.svg</normaloff>:/AuthorInformation/res/thatboy-text-vector.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>200</width>
      <height>40</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>411</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;班级：数嵌172&lt;/p&gt;&lt;p&gt;学号：202170638&lt;/p&gt;&lt;p&gt;擅长语言：C++&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="labelAvatar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>142</width>
      <height>200</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>142</width>
      <height>200</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>142</width>
      <height>200</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 5px solid #ABD7FA</string>
    </property>
    <property name="text">
     <string>This is my photo!!!</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>159</x>
      <y>80</y>
      <width>261</width>
      <height>96</height>
     </rect>
    </property>
    <widget class="QPushButton" name="btnGithub">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>48</y>
       <width>250</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>24</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align:left;</string>
     </property>
     <property name="text">
      <string>thatboy-echo</string>
     </property>
     <property name="icon">
      <iconset resource="AuthorInformation.qrc">
       <normaloff>:/AuthorInformation/res/github.png</normaloff>:/AuthorInformation/res/github.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnMail">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>24</y>
       <width>250</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>24</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align:left;</string>
     </property>
     <property name="text">
      <string>zhu.thatboy@outlook.com</string>
     </property>
     <property name="icon">
      <iconset resource="AuthorInformation.qrc">
       <normaloff>:/AuthorInformation/res/mail.png</normaloff>:/AuthorInformation/res/mail.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnWebsite">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>72</y>
       <width>250</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>24</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align:left;</string>
     </property>
     <property name="text">
      <string>https://thatboy.info</string>
     </property>
     <property name="icon">
      <iconset resource="AuthorInformation.qrc">
       <normaloff>:/AuthorInformation/res/boy.png</normaloff>:/AuthorInformation/res/boy.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnTelAddr">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>0</y>
       <width>250</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>24</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align:left;</string>
     </property>
     <property name="text">
      <string>13771042166, Jiangsu, China</string>
     </property>
     <property name="icon">
      <iconset resource="AuthorInformation.qrc">
       <normaloff>:/AuthorInformation/res/tel.png</normaloff>:/AuthorInformation/res/tel.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>171</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Dengxian','等线'; font-size:14pt;&quot;&gt;朱广锋&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Esc, F1</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="AuthorInformation.qrc"/>
 </resources>
 <connections/>
</ui>
